### ISSUE TITLE
Pagination::getPaginate() doesn't contain limit in result object

### ISSUE THREAD

**Issue by fizzka** [phalcon/cphalcon#3209]
It is not big problem, but it'll be more accurate to provide full information to result $page object.

**Comment by 7thcubic** [comment]
i presume you are talking about the total amount of items in the database/array right?

**Comment by fizzka** [comment]
I think we are talking about different subjects.

I mean _$limit_ from this construction:

```php
//from docs
$paginator = new \Phalcon\Paginator\Adapter\Model(
    array(
        "data" => $robots,
        "limit"=> 10,
        "page" => $currentPage
    )
);
```

When I call
```php
//from docs
$page = $paginator->getPaginate();
```

or pass reslults to view:

```php
$this->view->page = $paginator->getPaginate(); 
```

I can't access _"limit"_ ( => 10) property from _$page_.

Workaround I found is:

```php
//set limit
$limit = 10;
$paginator = new \Phalcon\Paginator\Adapter\Model(
    array(
        "data" => $robots,
        "limit"=> $limit,
        "page" => $currentPage
    )
);

$page = $paginator->getPaginate();
$page->limit = $limit;

$this->view->page = $page;
```

I suppose result must contain _limit_ value already ;)

**Comment by sergeyklay** [comment]
@fizzka 
I too use this workaround :)
also you can

```php
        $this->view->setVars([
            'page' => $page,
            'limit' => $limit
        ]);
```

**Comment by fizzka** [comment]
@sergeyklay 
where are many ways to get success, but I raised issue couse I'm sure that result _page_ object must be added with _limit_ =)

It is clean way - result object must incapsulate all neccessary data.

**Comment by 7thcubic** [comment]
I had suspected it was the manual limit. 

If it is that i suggest the work around. It isnt that much of a deal breaker.

Even if you used a global service to return the paginator object. The global service would already contain the limit information and could pass it back to you similarly. And since you initialize it on the fly, the limit info is something you already have, so it is my guess that it was intended to be this way. 

Again though extending pagination and adding a custom magic __get for limit isnt impossible. 

One scenario that i can think off when u pass the entire pagination object into view and you are lazy to pass limit too. I guess that would be needed. 

But most pagination applied to web normally allow the users to set the limit they want. 5/10/50 in the get parameter. So again there is redundancy.

**Comment by Green-Cat** [comment]
Implemented in the merged pull request #10020.
@phalcon @andresgutierrez could you close this issue please.
